addons/web/static/src/core/notifications/notification_service.js

```diff
@@ -50,6 +50,11 @@ export const notificationService = {
                 props,
                 onClose: options.onClose,
             };
+            for (const [notifId, notif] of Object.entries(notifications)) {
+                if (notif.props.message.toString() === notification.props.message.toString()) {
```

Comment:
I assume that the toString is for markup? You should have a testcase for this.
I'm also wondering if the `title` removal is on the way (I'm referring to the TODO-IPB in the template)? Because technically, you could have 2 notif with the same message but different titles ^^